submissions/peterIyiola/html/productPage-1.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title>Better Buys</title>
+    </head>
+<body>
+<nav>
+    <center><a href="#"><h1><b>Better Buys</b></h1></a>
```

Comment:
The `center` tag is not supported in HTML5. You need to remove it.

Reply:
okay